src/css/campaignPage.css

```diff
@@ -31,6 +31,10 @@ body {
     color: #3F5663;
 }
 
+#imgCol{
```

Comment:
where is this ID used?

Reply:
detail campaign page. the column inside the card for the campaign image. used to off set the image away from the left side to match the design